### DISCUSSION TITLE
How to enable SSL with Apache and Let's Encrypt?

### DISCUSSION THREAD

**Discussion by IlyaPetrovM** [directus/directus#7679] [Archive]
I have working https in 80 port (let's encrypt enabled).

I use VDS, debian 10, apache2, directus 9.0.0.rc90

Is it possible to "foward" https from 80 port to 8055 directus?

What type of config i need?

**Comment by rijkvanzanten** [comment]
Seems like you can use `ProxyPass` to set it up 👍🏻 

https://tecadmin.net/apache-frontend-proxy-nodejs/

**Reply by IlyaPetrovM** [reply]
@rijkvanzanten , that instruction works only for localhost sites, isn't it? 

I have my own domain t1553.ru 

###  /etc/apache2/sites-available/t1553.ru.conf
```
<VirtualHost *:80>
 ServerName t1553.ru

 ProxyRequests On
 ProxyPass / http://t1553.ru:8056/
 ProxyPassReverse / http://t1553.ru:8056/
</VirtualHost>
```

I set `PUBLIC_URL` to my external address to allow google OAuth to work (It is not working in other configurations)

### .env file of directus
```
## General

PORT=8056
PUBLIC_URL="http://t1553.ru:8056"
...
```

As a result I get that console output after loading page  t1553.ru : 
![image](https://user-images.githubusercontent.com/6277090/131706631-47301b86-1b84-48c7-84c7-30ba6e9599b4.png)

What am I doing wrong?

**Reply by rijkvanzanten** [reply]
> What am I doing wrong?

You're configuring a port proxy from port 80 on the public web (`<VirtualHost *:80>`) to local port 8056 on the server. This bit ` ProxyPass / http://t1553.ru:8056/` should be pointed to the locally running instance `http://localhost:8056/`. With that in place, your public URL in Directus itself will be port 80 (which is the default), so just `http://t1553.ru` 👍🏻

**Reply by IlyaPetrovM** [reply]
Amazing! It works! Thank you!
